loops.js

```diff
@@ -49,4 +64,14 @@
 // >> 7
 // >> 9
 // ==========================================
-
+for (i=0; i<10; i++) {
+    if (i>=0 && i<3) {
+        console.log (i);
+    }
+    if (i>2 && i<6) {
+        console.log(">" + i);
+    }
+    if (i>5 && i<10) {
+        console.log(">>" + i);
+    }
```

Comment:
Op regel 68 en 74 heb je een extra voorwaarde opgegeven die van zichzelf al waar is, i >= 0 op regel 68 en i < dan 10, daardoor check je een overbodige actie. Maakt nu niets uit, waarschijnlijk heeft de compiler het ook al door, het leest wel iets lastiger.